packages/modelviewer.dev/styles/docs.css

```diff
@@ -190,3 +172,35 @@ hr.solid {
 .bolden {
   font-weight: bold;
 }
+
+/* Smaller screen/ mobile */
+@media only screen and (max-width: 800px) {
+  :root {
+    --sidebar-width: 0;
```

Comment:
Looks like our descriptions are not justified to match the headings on mobile. Also, there are pretty big margins on both sides for mobile; seems like we could shrink those a bit.

Reply:
yeah, design wise, I wasn't sure what looked best